types/jsonpath-plus/jsonpath-plus-tests.ts

```diff
@@ -0,0 +1,4 @@
+const result = JSONPath.JSONPath({
+    path: "someXPath",
+    json: {},
```

Comment:
Need test for global usage:
Refer [this](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module) for more details.

Reply:
Thank you for pointing me to that helpful direction. I believe I've followed it in my latest push.